crates/edr_eth/src/transaction/request/eip4844.rs

```diff
@@ -103,3 +125,37 @@ impl rlp::Encodable for Eip4844TransactionRequest {
         s.append_list::<&[u8], &[u8]>(blob_hashes.as_slice());
     }
 }
+
+#[cfg(test)]
+pub(crate) mod tests {
+    use std::str::FromStr;
+
+    use revm_primitives::Address;
+
+    use super::*;
+    use crate::transaction::request::fake_signature::tests::test_fake_sign_properties;
+
+    fn dummy_request() -> Eip4844TransactionRequest {
+        // From https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/tx/test/eip4844.spec.ts#L68
+        Eip4844TransactionRequest {
+            chain_id: 0x28757b3,
+            nonce: 0,
+            max_priority_fee_per_gas: U256::from(0x12a05f200u64),
+            max_fee_per_gas: U256::from(0x12a05f200u64),
+            max_fee_per_blob_gas: U256::from(0xb2d05e00u64),
+            gas_limit: 0x33450,
+            to: Address::from_str("0xffb38a7a99e3e2335be83fc74b7faa19d5531243").unwrap(),
+            value: U256::from(0xbc614eu64),
+            input: Bytes::default(),
+            access_list: Vec::new(),
+            blob_hashes: vec![B256::from_str(
+                "0x01b0a4cdd5f55589f5c5b4d46c76704bb6ce95c0a8c09f77f197a57808dded28",
+            )
+            .unwrap()],
+        }
+    }
+
+    test_fake_sign_properties!();
+
+    // Hardhat doesn't support EIP-4844 yet, hence no test vector.
```

Comment:
There is a link to test vectors in the `signed/eip4844.rs` file. Maybe you can use that as a test vector here too.

Reply:
Thanks! The purpose of the fake signature test vectors is to make sure that the Hardhat implementation matches, so if there is no fake transaction Hardhat implementation for EIP-4844, I don't think it makes sense to add test vectors from a different repo. Updated the comment in an attempt to clarify.